Source/Core/VideoCommon/TextureDecoder_Common.cpp

```diff
@@ -434,6 +434,11 @@ static inline u32 MakeRGBA(int r, int g, int b, int a)
   return (a << 24) | (b << 16) | (g << 8) | r;
 }
 
+static inline u8 DXTBlend(u8 v1, u8 v2)
```

Comment:
This can be `static constexpr` or `static` instead of `static inline`. `inline` is generally unnecessary in the presence of static. Ditto for other one in the generic decoder.